modules/apps/segments/segments-web/src/main/resources/META-INF/resources/segments_experience_selector.jsp

```diff
@@ -28,6 +30,17 @@ SegmentsExperienceSelectorDisplayContext segmentsExperienceSelectorDisplayContex
 					<%= segmentsExperienceSelectorDisplayContext.getSelectedSegmentsExperienceName() %>
 				</span>
 
+				<%
+				boolean isSelectedExperienceActive = segmentsExperienceSelectorDisplayContext.isSelectedSegmentsExperienceActive(segmentsExperiencesJSONArray);
```

Comment:
Could we store segmentsExperiencesJSONArray in the segmentsExperienceSelectorDisplayContext., so we don't need to pass it as a parameter?